.github/workflows/mypy.yaml

```diff
@@ -45,10 +50,14 @@ jobs:
 
       - name: Run mypy
         run: |
-          prefix="python/lsst"
-          if [ ! -e "${prefix}" ]
+          prefix="${{ inputs.folders }}"
+          if [ -z "${prefix}" ]
           then
-            prefix="python"
+            prefix="python/lsst"
```

Comment:
I'm pretty sure that "python/lsst" is a historical artifact from when we had some problem with python namespaces in butler and now `mypy python` Just Works. In which case it might be clearer to remove this code completely and put `default: "python"` in the `folders` definition, making the default be the actual default rather than a proxy.